src/api/app/views/webui/package/show.html.haml

```diff
@@ -93,3 +93,10 @@
     = render partial: 'webui/request/delete_request_dialog', locals: { project: @project, package: @package }
   - if @package.develpackage
     = render partial: 'webui/request/change_devel_request_dialog', locals: { project: @project, package: @package }
+
+- content_for :ready_function do
+  :plain
+    $(document).ready(function() {
```

Comment:
:ready_function is already in the ready function

Reply:
I'm gonna remove this, but thanks

Reply:
I mean, this is left over stuff from a previous experiment